### PR TITLE
Moving to a global config. Removed ws and kafka configs.

### PR DESCRIPTION
**Pull request by Jason-RH**
@dehort This change adds a global config package. If you miss the "GetConsumer/GetProducer" type functions we can easily add those back.

### CODE REVIEW

File: cmd/gateway/main.go [hunk 1]
```diff
@@ -55,10 +65,10 @@ func main() {
 	apiSpecServer := api.NewApiSpecServer(apiMux, OPENAPI_SPEC_FILE)
 	apiSpecServer.Routes()
 
-	mgmtServer := api.NewManagementServer(cm, apiMux, wsConfig.ServiceToServiceCredentials)
+	mgmtServer := api.NewManagementServer(cm, apiMux, cfg.ServiceToServiceCredentials)
```
Comment (dehort) [inline cmd/gateway/main.go:68]: What do you think about passing the entire config object to the ManagementServer and JobReceiver constructors?

File: cmd/gateway/main.go [hunk 2]
```diff
@@ -55,10 +65,10 @@ func main() {
...
 	mgmtServer.Routes()
 
-	jr := api.NewJobReceiver(cm, apiMux, kw, wsConfig.ServiceToServiceCredentials)
+	jr := api.NewJobReceiver(cm, apiMux, kw, cfg.ServiceToServiceCredentials)
```
Comment (dehort) [inline cmd/gateway/main.go:71]: What do you think about passing the entire config object to the ManagementServer and JobReceiver constructors?

File: cmd/gateway/main.go (hunk 1 above)
Comment (Jason-RH) [reply cmd/gateway/main.go:68]: We can do that!

**Comment by Jason-RH** [comment]
@dehort :man_facepalming: on forgetting to push the new files. Also went ahead and updated job receiver and management to take the entire config object.

**Pull request merged**